all-tests/pom.xml

```diff
@@ -5,7 +5,7 @@
 	<parent>
 		<groupId>org.jboss.tools</groupId>
 		<artifactId>aerogear</artifactId>
-		<version>1.1.0-SNAPSHOT</version>
+		<version>1.2.0-SNAPSHOT</version>
```

Comment:
@dgolovin Without this fix, all-tests/pom.xml will fail in Jenkins because there's no 1.1.0 root pom available. Can you push this? We need it for the code freeze for Alpha1.